### ISSUE TITLE
Unable to extract m-25012018-123527.pth.tar

### ISSUE THREAD

**Issue by gambitier** [zoogzog/chexnet#7]
hi, I am trying to extract m-25012018-123527.pth.tar, but while doing so error occurred. I have tried it on both windows and ubuntu.

**Comment by zoogzog** [comment]
I don't think you are supposed to extract the model file at all. I am not even sure it is a tar archive at all, yet that is how networks are saved in various examples. For example: [imagenet](https://github.com/pytorch/examples/blob/master/imagenet/main.pyl) (line 267), [mnist](https://github.com/floydhub/save-and-resume/blob/master/pytorch_mnist_cnn.py)  (line 62). 

Here is the [documentation](http://pytorch.org/docs/0.3.1/notes/serialization.html#recommend-saving-models) about saving/loading networks.

In my code, I can load the saved model to continue training as presented below, where _checkpoint_ is the path to the _tar_ file.

`modelCheckpoint = torch.load(checkpoint)`
         ` model.load_state_dict(modelCheckpoint['state_dict'])`
            `optimizer.load_state_dict(modelCheckpoint['optimizer'])`

**Comment by hyattbaker** [comment]
Had the same question thanks @zoogzog
